src/test/java/org/sefglobal/scholarx/service/MentorServiceTest.java

```diff
@@ -85,7 +86,15 @@ void applyAsMentee_withValidData_thenReturnCreatedData()
             throws ResourceNotFoundException, BadRequestException {
         final Mentor mentor = new Mentor();
         mentor.setState(EnrolmentState.APPROVED);
-        mentor.setProgram(new Program());
+
+        Program program = new Program();
+        program.setState(ProgramState.MENTEE_APPLICATION);
+        mentor.setProgram(program);
+
+        Profile profile = new Profile();
+        profile.setId(profileId);
+        mentor.setProfile(profile);
```

Comment:
We don't need this part because we are not checking the profile id and mentor's profile